openssl/src/ssl/mod.rs

```diff
@@ -312,6 +321,22 @@ impl SslContext {
             })
     }
 
+    /// Specifies the private key
+    pub fn set_private_key(&mut self, key: &PKey) -> Option<SslError> {
+        wrap_ssl_result(
+            unsafe {
+                ffi::SSL_CTX_use_PrivateKey(*self.ctx, key.get_handle())
+            })
+    }
+
+    /// Check consistency of private key and certificate
+    pub fn check_private_key(&mut self) -> Option<SslError> {
```

Comment:
Yes, in deed. All other functions also use `wrap_ssl_result()` which returns `Option<SslError>`.
I think it's best (and consistent) if I modify `wrap_ssl_result()` to return `Result<(), SslError>`.
Do you agree?